CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## [v1.2.3](http://github.com/ndustrialio/contxt-sdk-js/tree/v1.2.3) (2019-06-05)
```

Comment:
I'm not sure if anyone else has a stronger opinion on this but I think this qualifies as a minor version bump due to just adding brand new functionality vs bug fixes.

Reply:
This is what lee told me last time...

Reply:
I don't think we've settled on any clear internal guidelines to what should be a patch bump vs a minor bump. In the past we've always just done a patch but we should probably set some clear guidelines sometime soon

Reply:
It should be a minor bump:
- MAJOR version when you make incompatible API changes,
- MINOR version when you add functionality in a backwards-compatible manner, and
- PATCH version when you make backwards-compatible bug fixes.

Reply:
We should probably think about how we're going to handle patching a minor version that we've already passed.

For example if `master` is on `1.4.1` and someone sees a bug in `1.2.2` what's the git/work flow to patch `1.2.2`?

Reply:
Probably just patch it once per major version - a minor release shouldn't cause any breaking changes so I don't see it as a problem to update the most recent version

Reply:
> We should probably think about how we're going to handle patching a minor version that we've already passed.
> 
> For example if `master` is on `1.4.1` and someone sees a bug in `1.2.2` what's the git/work flow to patch `1.2.2`?

Usually at that point you can branch off of the tagged change.